### DISCUSSION TITLE
How do I search and discover open source OPA bundles?

### DISCUSSION THREAD

**Discussion by nitrocode** [open-policy-agent/community#524] [OPA and Rego]
Recently I've been reading about running opa using a [bundle](https://www.openpolicyagent.org/docs/latest/management-bundles/) to help centralize policies in a semantically versioned repo.

My use case is to search for a public terraform opa bundle. I looked for an opa registry, similar to the terraform registry, but I do not believe one exists.

How do I discover opa bundles to use without having to craft my own custom rego policies?

---

My unideal workaround is [github search](https://github.com/search?q=+path%3A%2F%28%5E%7C%5C%2F%29%5C.manifest.*%24%2F+roots+-lang%3AJavaScript+&type=code)

**Comment by charlieegan3** [comment]
Hi @nitrocode, the bundles are intended to contain the policy that you have written, encoding your policy and data needed to evaluate decisions. There is some related discussion [here](https://github.com/open-policy-agent/opa/issues/3371) but I think the best place you can look is in the public policy repos: https://github.com/StyraInc/awesome-opa/tree/main#policy-packages
